tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -518,16 +432,17 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
             <param name="mismatch" value="-3" />
             <param name="gap_open" value="5"/>
             <param name="gap_ext" value="2"/>
-            <param name="ambiguous_letter" value="-3"/>
-            <output name="aligned_paired" file="test5_aligned.fasta" />
-            <output name="unaligned_paired" file="test5_unaligned.fasta" />
+            <param name="N" value="True"/>
+            <output name="aligned_forward" file="test5_aligned_forward.fasta" />
+            <output name="aligned_reverse" file="test5_aligned_reverse.fasta" />
+            <output name="output_bam" file="test5_bam.bam" compare="sim_size" delta="200" />
         </test>
-        <test>
+        <!--test expect_num_outputs="4">
```

Comment:
why do you take them out? not useful anymore?

Reply:
Yes sorry, I just commented in order to fix some of them. The problem is that when running locally, it is not possible to run all fo them at once because it stores a temporal file (/home/laptop/sortmerna/run/kvdb), which needs to be removed each time.